internal/serve/httphandler/forgot_password_handler_test.go

```diff
@@ -57,51 +72,77 @@ func Test_ForgotPasswordHandler(t *testing.T) {
 		SDPUIBaseURL: &uiBaseURL,
 	}
 	ctx := tenant.SaveTenantInContext(context.Background(), &tnt)
+	const email = "valid@email.com"
+	defaultUserRoles := []string{data.OwnerUserRole.String()}
+	user := &auth.User{
+		ID:    "userID",
+		Email: email,
+	}
+	const reCAPTCHAToken = "validToken"
+	defaultReqBody := fmt.Sprintf(`
+				{ 
+					"email": "%s",
+					"recaptcha_token": "%s"
+				}`, user.Email, reCAPTCHAToken)
 
 	t.Run("Should return http status 200 on a valid request", func(t *testing.T) {
-		requestBody := `
-		{ 
-			"email": "valid@email.com" ,
-			"recaptcha_token": "validToken"
-		}`
-
-		rr := httptest.NewRecorder()
-		req, err := http.NewRequestWithContext(ctx, http.MethodPost, url, strings.NewReader(requestBody))
-		require.NoError(t, err)
-
-		authenticatorMock.
-			On("ForgotPassword", req.Context(), "valid@email.com").
-			Return("resetToken", nil).
-			Once()
-		reCAPTCHAValidatorMock.
-			On("IsTokenValid", mock.Anything, "validToken").
-			Return(true, nil).
-			Once()
-
-		resetPasswordLink, err := urllib.JoinPath(uiBaseURL, "reset-password")
-		require.NoError(t, err)
+		usersRoles := make([][]string, 2)
+		// role cannot bypass reCAPTCHA
+		usersRoles[0] = []string{data.OwnerUserRole.String()}
+		// role can bypass reCAPTCHA
+		usersRoles[1] = []string{data.OwnerUserRole.String(), data.APIUserRole.String()}
```

Comment:
💡 This could be a little bit more readable IMO:
```suggestion
		usersRoles := [][]string{
			{data.OwnerUserRole.String()},                            // cannot bypass reCAPTCHA
			{data.OwnerUserRole.String(), data.APIUserRole.String()}, // API role can bypass reCAPTCHA
		}
```